### PR TITLE
Add Receptor Installation settings select

### PR DESCRIPTION
**Pull request by zkayyali812**
##### SUMMARY

This PR adds in 2 new settings - 
1. RECEPTOR_INSTALLATION_METHODS = ['release', 'package', 'local', 'container']
2. DEFAULT_RECEPTOR_INSTALLATION_METHOD = 'release'

`DEFAULT_RECEPTOR_INSTALLATION_METHOD` can be overridden by users to provide an alternate default

##### ISSUE TYPE

 - New or Enhanced Feature

##### COMPONENT NAME

 - UI
 - Docs
 - Other

##### AWX VERSION

```
➜  awx git:(zk-add-receptor-settings) ✗ make version     
make: *** No rule to make target `version'.  Stop.
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: awx/main/management/commands/host_metric.py [hunk 1]
```diff
@@ -35,15 +35,7 @@ def host_metric_queryset(self, result, offset=0, limit=BATCHED_FETCH_COUNT):
...
-            result.values(
-                'id',
-                'date',
-                'license_consumed',
-                'license_capacity',
-                'hosts_added',
-                'hosts_deleted',
-                'indirectly_managed_hosts',
-            ).order_by(
+            result.values('id', 'date', 'license_consumed', 'license_capacity', 'hosts_added', 'hosts_deleted', 'indirectly_managed_hosts',).order_by(
```
Comment (TheRealHaoLiu) [inline awx/main/management/commands/host_metric.py]: revert this change i think your editor setting is conflicting with black

**Comment by TheRealHaoLiu** [comment]
```
/api/v2/instances/                   | instance                   | receptor_installation_method                              |  X  |        | Failed, option mismatch
```
awx-collection completeness check failed

File: awx/settings/defaults.py [hunk 2]
```diff
@@ -967,6 +967,10 @@
 # K8S only. Use receptor_log_level on AWX spec to set this properly
 RECEPTOR_LOG_LEVEL = 'info'
 
+RECEPTOR_INSTALLATION_METHODS = ['release', 'package', 'local', 'container']
```
Comment (chrismeyersfsu) [inline awx/settings/defaults.py:970]: I don’t see this variable used. Can we delete it?

Comment (TheRealHaoLiu) [reply awx/settings/defaults.py:970]: would be used by the UI to populate the selection drop down 

also somewhere we want to be able to limit what options are "exposed" for example downstream "local" and "release" wont be available

File: awx/main/migrations/0189_instance_receptor_installation_method.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            model_name='instance',
+            name='receptor_installation_method',
+            field=models.CharField(
+                choices=[
+                    ('release', 'Install from GitHub release'),
+                    ('package', 'Install from RPM package'),
+                    ('local', 'Install from local source'),
+                    ('container', 'Install via container'),
+                ],
+                default='release',
```
Comment (TheRealHaoLiu) [inline awx/main/migrations/0189_instance_receptor_installation_method.py:22]:
```suggestion
                default='package',
```

**Review by fosterseth — APPROVED** [review]

**Review by jbradberry — APPROVED** [review]
